Tasks/CocoaPodsV0/task.json

```diff
@@ -17,8 +17,8 @@
     "minimumAgentVersion": "2.182.1",
     "version": {
         "Major": 0,
-        "Minor": 201,
-        "Patch": 1
+        "Minor": 204,
+        "Patch": 0
```

Comment:
Could you please commit these changes for the `task.loc.json` file as well?

Reply:
added